src/box/sql/tokenize.c

```diff
@@ -624,7 +624,7 @@ struct sql_trigger *
 sql_trigger_compile(const char *sql)
 {
 	struct Parse parser;
-	sql_parser_create(&parser, default_flags);
+	sql_parser_create(&parser, SQL_DEFAULT_FLAGS);
```

Comment:
Why don't you use the `sql_flags()` function here and in other places above? I'd expect you use the same default flags everywhere. If `sql_flags()` are solely for session, the function should be renamed appropriately, e.g. to `sql_default_session_flags()` or something like this.

Reply:
We decided that SEQSCAN changes the behavior of SELECT and should not affect other statements, so there is no need to call `sql_flags()`. Renamed the function to `sql_default_session_flags()`.

Reply:
What "other statements"?

Reply:
Statement to create `SQL_EXPR` function, statement to create trigger and statement to create view.